### PR TITLE
99.32% val accuracy with a deeper CNN.

### LINKED ISSUE #15: Create a more complicated convolutional network
So far we've only done a baseline CNN. Let's do something more complicated and see how good we can get without extending our data or reaching out to a pretrained network.

### PR DESCRIPTION
**Pull request by MatthewChatham**
Closes #15.

Achieved 99.32% validation accuracy, even after changing the model search strategy to be based on testing data instead of validation data.

### CODE REVIEW

**Comment by MatthewChatham** [comment]
Next up: #5 using [this](http://cs231n.github.io/understanding-cnn/)/
